src/legacy/Routes.tsx

```diff
@@ -610,8 +621,9 @@ class RouteApp extends React.Component<any, any> {
 
               <Route
                 path={routes.LOADING}
-                render={() => (
+                render={p => (
```

Comment:
I don't know what `p` is, use more descriptive name

Reply:
p: prop of route when rendering @adrianobetkal 
It's so easy for understand for developers.

Reply:
The good practice is to never user letters or shorts. Any argument, const, let, etc names should be descriptive even if you think `It's so easy for understand for developers.`